azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/DeployMojo.java

```diff
@@ -203,9 +203,6 @@ private void deploy(IWebAppBase target, WebAppConfig config) throws AzureExecuti
         }
         try {
             Log.info(String.format(DEPLOY_START, config.getAppName()));
-            if (isStopAppDuringDeployment()) {
```

Comment:
This is a shared configuration in v1/v2 which support stop app before deployment, do we need to remove it?

Reply:
add it back.